recipes/qt/5.x.x/conanfile.py

```diff
@@ -90,10 +67,10 @@ class QtConan(ConanFile):
         "gui": [True, False],
         "widgets": [True, False],
 
-        "device": "ANY",
-        "cross_compile": "ANY",
-        "sysroot": "ANY",
-        "config": "ANY",
+        "device": ["ANY"],
+        "cross_compile": ["ANY"],
+        "sysroot": ["ANY"],
+        "config": ["ANY"],
```

Comment:
You can add `None` to this list to allow using `None` as a default value below: https://docs.conan.io/1/migrating_to_2.0/recipes.html?highlight=any#id9

```suggestion
        "device": [None, "ANY"],
        "cross_compile": [None, "ANY"],
        "sysroot": [None, "ANY"],
        "config": [None, "ANY"],
```